bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAlarmHandler.java

```diff
@@ -229,27 +233,29 @@ private void updateZoneState(WhatAlarm w) {
     }
 
     private void updateZoneAlarm(WhatAlarm w) {
+        LocalDateTime now = LocalDateTime.now(ZoneId.systemDefault());
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm:ss");
         switch (w) {
             case ZONE_ALARM_INTRUSION:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_INTRUSION));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_INTRUSION + " " + now.format(format)));
```

Comment:
timestamp should not be concatenated to the `alarm` channel, but returned in a new `timestamp` channel. This is not a good design. See explanation in previous comment